### ISSUE TITLE
We need to have Security Audit for long term grow up

### ISSUE THREAD

**Issue by kineita** [horizontalsystems/unstoppable-wallet-android#7426]
Our app is very good about performance with many best practices from Google, Apple to bring this app to the world. But we need to have security from many sources to check security holes in our app 
I see Certik, ISACA, ... for example, thank you so much

**Comment by kineita** [comment]
Yes, it is correct with commercial project. We are open source project, we will have many support from other sides and many resources.

**Comment by 48-mq** [comment]
Hate to burst your bubble @kineita but this is very much a commercial project, even tough it is MIT-licensed. [Horizontal Systems](https://horizontalsystems.io/) is a for-profit company based in Hong Kong.

And while an external audit would only help with commercializing this project, it would almost certainly come at a substantial cost.

**Comment by kineita** [comment]
Hi @48-mq, but this product don't have any commercial parts or coding, currently it is full open source 100% (not half half like other projects). So I think this project can be considered like real open source project in this time
